Makefile

```diff
@@ -36,11 +33,7 @@ start: build
 stop:
 	docker-compose down
 
-publish: build
-	docker push ${SERVICE_IMG}:${TAG}
-	docker push ${STATIC_IMG}:${TAG}
-
-deploy: publish
+deploy:
 	terraform remote config \
```

Comment:
Although my changes to `TAG` ended up impacting the `deploy` portion of this Makefile, I don't think  adding `scripts/infra` was in the scope of the issue so I haven't added that script.